Invalid argument error in Windows path

I got the following errors while resolving an issue

```
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
[Errno 22] Invalid argument: '/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java'
```

I can imagine the error comes from the first "/" in the path

[extension-Konveyor-Analyzer.log](https://github.com/user-attachments/files/18963629/extension-Konveyor-Analyzer.log)
[kai-rpc-server.log](https://github.com/user-attachments/files/18963633/kai-rpc-server.log)
[kai-analyzer-server.log](https://github.com/user-attachments/files/18963632/kai-analyzer-server.log)

[analysis_20250225T105901.json](https://github.com/user-attachments/files/18963959/analysis_20250225T105901.json)
[solution_20250225T111024.json](https://github.com/user-attachments/files/18963958/solution_20250225T111024.json)

**Env:**
* Windows 11 Pro
* Extension v0.0.12
* Get Solution Max Effort: High

It appears to me the issue is that the parsing of the files for the maven task runner is not getting the correct filepaths for windows:
```
BUG - 2025-02-25 12:05:10,076 - kai.kai.reactive_codeplanner.task_runner.compiler.maven_validator - ThreadPoolExecutor-1_0 - [maven_validator.py:628 - deduplicate_errors()] - adding collapsed_error: SymbolNotFoundError<loc=/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java:11:2, message=cannot find symbol>(priority=2(2), depth=0, retries=0)-lines:None -- for key: /c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java-<class 'kai.reactive_codeplanner.task_runner.compiler.maven_validator.SymbolNotFoundError'>-class Stateless
DEBUG - 2025-02-25 12:05:10,077 - kai.kai.reactive_codeplanner.task_runner.compiler.maven_validator - ThreadPoolExecutor-1_0 - [maven_validator.py:47 - run()] - determining whether maven cache should be set: build_errors: False, dependency_errors: False, catchall_errors: False, compilation_errors: True, cond: []
DEBUG - 2025-02-25 12:05:10,077 - kai.kai.reactive_codeplanner.task_runner.compiler.maven_validator - ThreadPoolExecutor-1_0 - [maven_validator.py:59 - run()] - Setting the maven cache with 1 compilation errors
DEBUG - 2025-02-25 12:05:10,078 - kai.kai.reactive_codeplanner.task_manager.task_manager - Thread-1 - [task_manager.py:173 - run_validators()] - Validator result: ValidationResult(passed=False, errors=[SymbolNotFoundError(priority=2, depth=0, parent=None, children=[], retry_count=0, max_retries=3, result=None, file='/c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java', line=11, column=2, message='cannot find symbol', details=['symbol: class Stateless'], parse_lines='[ERROR] /c:/Users/Brugui/PycharmProjects/kai-ci/coolstore/src/main/java/com/redhat/coolstore/service/OrderService.java:[11,2] cannot find symbol\nsymbol: class Stateless', missing_symbol='class Stateless', symbol_location=None)])
```

--- a/kai/reactive_codeplanner/task_runner/compiler/maven_validator.py
+++ b/kai/reactive_codeplanner/task_runner/compiler/maven_validator.py
@@ -2,6 +2,7 @@
 
 import re
 import subprocess  # trunk-ignore(bandit/B404)
+import sys
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
 from functools import cached_property
@@ -90,6 +91,12 @@ class MavenCompilerError(ValidationError):
 
     def compiler_error_message(self) -> str:
         return self.message
+
+    def __post_init__(self) -> None:
+        if sys.platform == "win32":
+            regex = re.compile(r"^/[a-zA-Z]:")
+            if re.match(regex, self.file):
+                self.file = self.file.removeprefix("/")
 
 
 class CollapsedMavenCompilerError(ABC, MavenCompilerError):
@@ -586,7 +593,7 @@ def catchall(output: str) -> OtherError:
     file_path_pattern = re.compile(r"(/[^:\s]+)")
     file_path_matches = file_path_pattern.findall(output)
     file_path = file_path_matches[0] if file_path_matches else "unknown file"
-    return OtherError(
+    error = OtherError(
         file=file_path,
         line=-1,
         column=-1,
@@ -594,6 +601,7 @@ def catchall(output: str) -> OtherError:
         details=[output],
         parse_lines=output,
     )
+    return error
 
 
 def deduplicate_errors(errors: list[MavenCompilerError]) -> list[MavenCompilerError]:

